platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -21,19 +39,6 @@ fi
 rm -f /var/run/sswsyncd/sswsyncd.socket
```

Comment:
we have it here.

Reply:
I missed that. I made a new commit to remove that line, keeping this PR open to simple reorganize the file.